modules/update-service-create-service-cli.adoc

```diff
@@ -0,0 +1,75 @@
+[id="update-service-create-service-cli"]
+= Creating an Update Service using the CLI
+include::modules/common-attributes.adoc[]
+
+You can use the {product-title} CLI to create an Update Service using the {product-title} Update Service operator.
+
+== Procedure
+
+To create an Update Service using the {product-title} CLI:
+
+.. Configure the Update Service target namespace:
++
+[source,terminal]
+----
+$ NAMESPACE=FIXME-default-operand-namespace
+----
++
+This must match `targetNamespaces` from the OperatorGroup.
+
+.. Configure the name of the Update Service:
++
+[source,terminal]
+----
+$ NAME=example
+----
+
+.. Configure the local registry and repository for the release images:
++
+[source,terminal]
+----
+$ RELEASE_IMAGES=registry.example.com/openshift/release
+----
+
+.. Configure the local pullspec for the graph-data image:
++
+[source,terminal]
+----
+$ GRAPH_DATA_IMAGE=registry.example.com/openshift/graph-data:latest
+----
+
+.. Create an Update Service YAML file (for example, `update-service.yaml`):
++
+[source,terminal]
+----
+$ oc -n "${NAMESPACE}" create -f - <<EOF
+apiVersion: updateservice.openshift.io/v1
+kind: UpdateService
+metadata:
+  name: ${NAME}
+spec:
+  replicas: 2
+  releaseImages: ${RELEASE_IMAGES}  # FIXME: aspirational
+  graphDataImage: ${GRAPH_DATA_IMAGE}
+EOF
+----
+
+. Verify the Update Service:
+
+.. Wait for a policy engine route, polling until:
++
+[source,terminal]
+----
+$ POLICY_ENGINE_GRAPH_URI="$(oc -n "${NAMESPACE}" get -o jsonpath='{.status.policyEngineURI}/api/upgrades_info/v1/graph}{"\n"}' updateservice example-name)"
+----
++
+succeeds.
```

Comment:
> Can you show sample successful output?

There is no output.  We could create some, but the command's exit code seems more reliable.  I can probably wrap this up in a `while` loop or some such so the user doesn't have to manually poll...

Reply:
If there's no output, that's fine, but it seemed like it might be a good chance for the user to know what success looks like. Not having to manually poll would be a better user experience.

Reply:
I've pushed e7474466c -> 6d253db05, pivoting from asking the user to poll to giving them a POSIX-shell command that will handle the polling for them.